chartpress.py

```diff
@@ -322,11 +340,10 @@ def build_image(
         "<context_path>/Dockerfile".
     build_args (dict, optional):
         Dictionary of docker build arguments.
-    extra_options (dict, optional):
-        Dictionary of other docker build options to use. Each key should be a
-        valid option to docker build, like "ssh" for the "--ssh" option. For 
-        flags like "--ssh" which take a key-value pair, make sure to use the
-        "key=value" syntax and not the "key value" syntax.
+    extra_build_command_options (list, optional):
+        List of other docker build options to use. Each item can be either a
+        string with the option name (e.g. "rm") or a dictionary with the key
+        and value (e.g. "label: 'maintainer=octocat'").
```

Comment:
A dict is still mentioned here

Reply:
Thanks -- I looked for similar errors in the docstrings and fixed those as well.